### PR TITLE
Add some emoji spinners

### PR DESCRIPTION
**Pull request by nathanhoad**
* the `weather` spinner animates the sun going behind a cloud and then rain
* the `christmas` spinner flashes Christmas lights on a tree

### CODE REVIEW

File: spinners.json [hunk 1]
```diff
@@ -835,5 +835,40 @@
...
 		]
+	},
+	"weather": {
+		"interval": 100,
+    "frames": [
+      "☀️ ",
+      "☀️ ",
+      "☀️ ",
+      "🌤 ",
+      "⛅️",
```
Comment (sindresorhus) [inline spinners.json]: You forgot a space here.

File: spinners.json [hunk 2]
```diff
@@ -835,5 +835,40 @@
 			"p",
 			"b"
 		]
+	},
+	"weather": {
+		"interval": 100,
+    "frames": [
```
Comment (sindresorhus) [inline spinners.json]: Incorrect indentation.

File: spinners.json (hunk 1 above)
Comment (nathanhoad) [reply spinners.json]: That is on purpose - that emoji has a width of two spaces by itself.

Comment (sindresorhus) [reply spinners.json]: I can reproduce that on Terminal.app (macOS), but on iTerm.app they all have the same width. I think the Terminal.app behavior is incorrect. It makes no sense that just this character is different width from all the other ones.

File: spinners.json [hunk 3]
```diff
@@ -835,5 +835,40 @@
 			"p",
 			"b"
 		]
-	}
+	},
+  "weather": {
+    "interval": 100,
+    "frames": [
```
Comment (sindresorhus) [inline spinners.json]: The indentation is still incorrect. It should be the same as the rest of the file (tab).

**Comment by sindresorhus** [comment]
Great! Thank you :)

**Pull request merged**